src/components/Cart/Content.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+import { spacingLarge } from '../../styles/designTokens';
+import BaseContent from '../Content';
+
+const Content = styled(BaseContent)`
```

Comment:
I would go with `ContentBase` since above you use similar convention for `Link` -> `LinkBase`.